forui/lib/src/theme/themes.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+
+import 'package:forui/forui.dart';
+
+/// The Forui themes.
+extension FThemes on Never {
+
+  /// The light and dark variants of the [Zinc](https://ui.shadcn.com/themes) theme.
+  static final zinc = (
+    light: FThemeData.inherit(
+      font: FFont(),
+      colorScheme: const FColorScheme(
+        background: Color(0xFFFFFFFF),
+        foreground: Color(0xFF09090B),
+        primary: Color(0xFF18181B),
+        primaryForeground: Color(0xFFFAFAFA),
+        secondary: Color(0xFFF4F4F5),
+        secondaryForeground: Color(0xFF18181B),
+        muted: Color(0xFFF4F4F5),
+        mutedForeground: Color(0xFF71717A),
+        destructive: Color(0xFFEF4444),
+        destructiveForeground: Color(0xFFFAFAFA),
+        border: Color(0xFFE4E4E7),
+      ),
+      style: FStyle(
+        borderRadius: BorderRadius.circular(8),
```

Comment:
Should we just add a default value for `borderRadius` on `FStyle` so we don't need to keep repeating it.